faq update: windows LNK1112 error (#415)
ponylang/ponylang-website: content/faq/index.md

ponyc -Dopenssl_1.1.0
```

### On Windows I get "fatal error LNK1112: module machine type 'x86' conflicts with target machine type 'x64'"

Only 64-bit Windows is supported.

> This error occurs when ponyc is compiled in Developer Command Prompt that is launched from 32-bit Visual Studio. 

Reviewer: I would change the word order a bit.
```suggestion
This error occurs when ponyc is compiled in a 32-bit Visual Studio Developer Command Prompt.
```

Author: changed.

> Make sure you're running pure cmd.exe/powershell.exe, that is not contaminated with VS's 32-bit env variables.

Reviewer: I would use more neutral language here.

```suggestion
Make sure you're running a cmd.exe/powershell.exe that does not include 32-bit VS environment variables.
```

Author: changed.
